include/dsAudio.h

```diff
@@ -1434,6 +1237,8 @@ dsError_t  dsEnableAudioPort(intptr_t handle, bool enabled);
 /**
  * @brief Enables or Disables MS12 DAPV2 and DE feature
  * 
+ * For source devices,this function returns dsERR_OPERATION_NOT_SUPPORTED always.
+ *
```

Comment:
Can we please check if this API is actually used or not?

Reply:
dsEnableMS12Config() is not used in sink and source devices

Reply:
Ok if we are putting a note that the API is deprecated.  Will remove it after internal review.